product_code/lagom/deploy/services/matriculation.yaml

```
@@ -0,0 +1,71 @@
+apiVersion: "apps/v1"
+kind: Deployment
+metadata:
+  namespace: uc4-lagom
+  name: matriculation
+spec:
+  replicas: 1
+
+  selector:
+    matchLabels:
+      app: matriculation
+
+  template:
+    metadata:
+      namespace: uc4-lagom
+      labels:
+        app: matriculation
+    spec:
+      containers:
+        - name: matriculation
+          image: "uc4official/matriculation_service:latest"
+          imagePullPolicy: Always
+          env:
+            - name: JAVA_OPTS
+              value: "-Xms256m -Xmx256m -Dconfig.resource=prod-application.conf"
+            - name: APPLICATION_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: hl-course-application-secret
```

```suggestion
                  name: matriculation-application-secret
```
Use the matriculation service secret